Source/WebKit/UIProcess/Gamepad/mac/UIGamepadProviderMac.mm

```diff
@@ -39,7 +39,8 @@
 WebPageProxy* UIGamepadProvider::platformWebPageProxyForGamepadInput()
 {
     ASSERT(hasProcessPrivilege(ProcessPrivilege::CanCommunicateWithWindowServer));
-    auto responder = [[NSApp keyWindow] firstResponder];
+    // This is a safer cpp false positive (<rdar://problem/161068288>).
+    SUPPRESS_UNRETAINED_ARG auto responder = [[NSApp keyWindow] firstResponder];
```

Comment:
keyWindow would be a false positive, but first responder would not be. So we should only suppress if the warning is about keyWindow.